craft_parts/sources/git_source.py

```diff
@@ -80,13 +82,19 @@ class GitSource(SourceHandler):
     """
 
     source_model = GitSourceModel
-    command: ClassVar[str] = "git"
+
+    @staticmethod
+    def command() -> str:
```

Comment:
if you are changing the signature for this, can we not use the new method this wraps directly instead?

Reply:
sure
done